versioned_sidebars/version-v1-sidebars.json

```diff
@@ -195,30 +195,20 @@
               "label": "Integrate with Okta",
               "id": "guides/integrate-with-okta"
             },
-            {
-              "type": "doc",
-              "label": "Integrate with KeyCloak",
-              "id": "guides/integrate-with-keycloak"
-            },
-            {
-              "type": "doc",
-              "label": "Integrate with Curity",
-              "id": "guides/integrate-with-curity"
-            },
             {
               "type": "doc",
               "label": "Integrate with Azure AD B2C",
               "id": "guides/integrate-with-azureb2c"
             },
             {
               "type": "doc",
-              "label": "Integrate with Beyond Identity Workforce",
-              "id": "guides/integrate-with-bi-workforce"
+              "label": "Integrate with KeyCloak",
+              "id": "guides/integrate-with-keycloak"
             },
             {
               "type": "doc",
-              "label": "Integrate with AWS Cognito",
-              "id": "guides/integrate-with-cognito"
+              "label": "Integrate with Curity",
+              "id": "guides/integrate-with-curity"
```

Comment:
Is there a best practice around the ordering of these guides?

Reply:
@Will5 it depends.  We can do them alphabetically or in the order of most frequently used (or most popular).  If there's no rhyme or reason for a particular order, the default is alphabetically.

Reply:
Ok, that makes sense; I usually prefer alphabetically as well, unless there is a rhyme or reason for a particular order.